optaweb/image.yaml

```diff
@@ -0,0 +1,172 @@
+    - name: "JAVA_MAX_INITIAL_MEM"
+      example: "4096"
+      description: "The maximum size of the initial heap memory, if the calculated default initial heap is larger then it will be capped at this value.  The default is 4096 MB."
+    - name: "JAVA_CORE_LIMIT"
+      example: "2"
+      description: "Core limit as described in https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt.  Used to configure the number of GC threads and parallelism for ForkJoinPool.  Defaults to container core limit."
+    - name: "JAVA_DIAGNOSTICS"
+      example: "true"
+      description: "Set this to get some diagnostics information to standard output when things are happening. **Disabled by default.**"
+    - name: "GC_MIN_HEAP_FREE_RATIO"
+      example: "20"
+      description: "Minimum percentage of heap free after GC to avoid expansion."
+    - name: "GC_MAX_HEAP_FREE_RATIO"
+      example: "40"
+      description: "Maximum percentage of heap free after GC to avoid shrinking."
+    - name: "GC_TIME_RATIO"
+      example: "4"
+      description: "Specifies the ratio of the time spent outside the garbage collection (for example, the time spent for application execution) to the time spent in the garbage collection."
+    - name: "GC_ADAPTIVE_SIZE_POLICY_WEIGHT"
+      example: "90"
+      description: "The weighting given to the current GC time versus previous GC times."
+    - name: "GC_MAX_METASPACE_SIZE"
+      example: "100"
+      description: "The maximum metaspace size."
+    - name: "CUSTOM_INSTALL_DIRECTORIES"
+      example: "custom,shared"
+      description: "A list of comma-separated directories used for installation and configuration of artifacts for the image during the S2I process."
+    - name: "CONTAINER_HEAP_PERCENT"
+      example: "0.5"
+      description: "Deprecated.  See JAVA_MAX_MEM_RATIO."
+    - name: "INITIAL_HEAP_PERCENT"
+      example: "0.5"
+      description: "Deprecated.  See JAVA_INITIAL_MEM_RATIO."
+    - name: "PROBE_DISABLE_BOOT_ERRORS_CHECK"
+      example: "true"
+      description: "Disable the boot errors check in the probes."
+    - name: "ENABLE_ACCESS_LOG"
+      example: "true"
+      description: "Enable the Access Log."
+    - name: "ENABLE_JSON_LOGGING"
+      example: "true"
+      description: "Enable JSON-formatted logging"
+    - name: "OPTAWEB_ROSTERING_PERSISTENCE_DIALECT"
+      example: "org.hibernate.dialect.MySQL5Dialect"
+      description: "server persistence dialect "
+    - name: "OPTAWEB_ROSTERING_PERSISTENCE_DATASOURCE"
+      example: "java:jboss/datasources/dbe"
+      description: "server persistence datasource for persistence.xml "
+    - name: "OPTAWEB_GENERATOR_TIMEZONE_ID"
+      example: "America/New_York"
+      description: "Keep in sync with https://github.com/kiegroup/optaweb-employee-rostering/blob/master/employee-rostering-docs/src/main/asciidoc/Introduction/SystemProperties/SystemProperties-section.adoc "
```

Comment:
The description should be `The time zone ID for the automatically generated tenants. For example America/New_York. This defaults to the system default Zone ID.`

Ideally, there is a yml comment line above it, so it ends up like this:

```
    - name: "OPTAWEB_GENERATOR_TIMEZONE_ID"
      example: "America/New_York"
      # Keep in sync with https://github.com/kiegroup/optaweb-employee-rostering/blob/master/employee-rostering-docs/src/main/asciidoc/Introduction/SystemProperties/SystemProperties-section.adoc
      description: "The time zone ID for the automatically generated tenants. For example America/New_York. This defaults to the system default Zone ID."
```